test/integration/targets/ansible-galaxy-collection-scm/tasks/scm_dependency_deduplication.yml

```diff
@@ -30,16 +49,14 @@
 
 - assert:
     that:
-      - command.stdout_lines | length == 9
-      - command.stdout_lines[0] == "Starting galaxy collection install process"
-      - command.stdout_lines[1] == "Process install dependency map"
-      - command.stdout_lines[2] == "Starting collection install process"
-      - "'namespace_1.collection_1' in command.stdout_lines[3]"
-      - "'namespace_1.collection_1' in command.stdout_lines[4]"
-      - "'namespace_1.collection_1' in command.stdout_lines[5]"
-      - "'namespace_2.collection_2' in command.stdout_lines[6]"
-      - "'namespace_2.collection_2' in command.stdout_lines[7]"
-      - "'namespace_2.collection_2' in command.stdout_lines[8]"
+      - command.stdout_lines | length == 12
+      - command.stdout_lines[5] == "Starting collection install process"
+      - "'namespace_1.collection_1' in command.stdout_lines[6]"
+      - "'namespace_1.collection_1' in command.stdout_lines[7]"
+      - "'namespace_1.collection_1' in command.stdout_lines[8]"
+      - "'namespace_2.collection_2' in command.stdout_lines[9]"
+      - "'namespace_2.collection_2' in command.stdout_lines[10]"
+      - "'namespace_2.collection_2' in command.stdout_lines[11]"
```

Comment:
This may potentially be order-dependent aka flaky, like the first test in this file.

Reply:
I was right. This happened again: https://dev.azure.com/ansible/ansible/_build/results?buildId=4597&view=logs&j=a00789c5-8412-587b-63fc-93dbdf0cd470&t=e049f74a-b955-5412-bd69-ba41533160f3&l=9242